specification/marketplacecatalog/data-plane/Search/preview/2022-09-25-preview/search.json

```diff
@@ -0,0 +1,2229 @@
+{
+  "x-generator": "NSwag v13.15.10.0 (NJsonSchema v10.6.10.0 (Newtonsoft.Json v13.0.0.0))",
+  "swagger": "2.0",
+  "info": {
+    "title": "Microsoft Marketplace Search API",
+    "version": "2022-09-25-preview"
+  },
+  "host": "catalogapi.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/search": {
+      "get": {
+        "tags": [
+          "Public-Search"
+        ],
+        "summary": "Get a list of azure marketplace catalog offers and total count and facets",
+        "operationId": "Search_Get",
```

Comment:
This operationId is odd. What is the intent? If you intend to generate client libraries from this, it will be very odd code?

Reply:
@johanste /search is not part of this PR, it was already reviewed in the previous PR but anyway, this endpoint is intended to get search results